Adds `ember-select-light` v2.0.0 release to embertimes-167 (#808)
ember-learn/ember-blog: source/2021-11-06-the-ember-times-issue-167.md

---
title: The Ember Times - Issue No. 167
author: Ava Wroten, Chris Ng, the crowd
tags: Recent Posts, Newsletter, A11y, Accessibility, Ember.js Times, Ember Times, 2020
alias : "blog/2020/11/06-the-ember-times-issue-167.html"
responsive: true
---

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹

> New Release and APIs for ember-a11y-testing 🎉, New Octane ready release of the a11y focused ember-select-light 🎉

Reviewer: We like to allow one line per news item to help with organization.

```suggestion
New Release and APIs for ember-a11y-testing 🎉,
New Octane ready release of the a11y focused ember-select-light 🎉,
```

---

## [Release 2.0.x of ember-select-light 🎉](https://github.com/ember-a11y/ember-select-light/releases/tag/v2.0.0)

> [Ava Wroten (@hergaiety)](https://www.wroten.me/) released 2.0.0 of [ember-select-light](https://github.com/ember-a11y/ember-select-light/) moving the addon into the [ember-a11y org](https://github.com/ember-a11y) (join us on [discord in #dev-ember-a11y](https://discord.gg/xBzFBMYv)) and making it **Octane ready**.

Reviewer: What do you think of breaking the sentence into two?

```suggestion
[Ava Wroten (@hergaiety)](https://www.wroten.me/) released 2.0.0 of [ember-select-light](https://github.com/ember-a11y/ember-select-light/). You will find that the addon has moved to the [ember-a11y org](https://github.com/ember-a11y) (join us on [discord in #dev-ember-a11y](https://discord.gg/xBzFBMYv)) and is **Octane ready**.
```

> **ember-select-light is a simple and accessible `<select>` dropdown component** ✨. With a simple API you can pass in a flat or nested array of options or yield out to your own manually handled options template code. This component pairs nicely with the latest [ember-component-patterns for Select Elements](https://emberjs-1.gitbook.io/ember-component-patterns/form-components/select-element).

Reviewer: Right now, we're having issues with Travis CI not running fast enough. Can you run `alex` on this file? I think it may ask you to use another word besides "simple" (you can make an exception if you prefer).

```bash
# If you don't want to install alex globally, I think you can try:
npx alex source/2021-11-06-the-ember-times-issue-167.md
```

Reviewer: To help with parsing the sentences, maybe:

```suggestion
**ember-select-light is a simple and accessible `<select>` dropdown component** ✨. With a simple API you can pass in a flat or nested array of options, or yield out to your template code that handles options. The component pairs nicely with the latest [ember-component-patterns for Select Elements](https://emberjs-1.gitbook.io/ember-component-patterns/form-components/select-element).
```

> With the new 2.0.0 drops support for older versions of Node and Ember. Node v10 LTS and Ember.js v3.16 LTS are suggested. Some breaking syntax changes (see the [upgrade guide](https://github.com/ember-a11y/ember-select-light/blob/main/UPGRADE.md)) Lastly, documentation and plenty of housekeeping was done leading up to this release.

Reviewer: I wasn't sure what you meant by housekeeping. I think we could remove the part of the sentence for clarity?

```suggestion
The 2.0.0 release drops support for older versions of Node and Ember. At minimum, Node v10 LTS and Ember.js v3.16 LTS are suggested. It also introduces some breaking syntax changes (see the [upgrade guide](https://github.com/ember-a11y/ember-select-light/blob/main/UPGRADE.md)). Lastly, documentation has been updated.
```
